### DISCUSSION TITLE
How to change the coefficient of PAR

### DISCUSSION THREAD

**Discussion by liujia96** [NGEET/fates#1006] [Q&A]
I want to design an experiment to change the total radiation without changing the photosynthesis of vegetation, so if I want to adjust the parameters of PAR, which code should I modify.

**Comment by rgknox** [comment]
The radiation scattering over the depth of the canopy is not impacted by the magnitude of the downwelling radiation.  So if you increased or decreased that magnitude, I think that the relative profiles of trasmittance and absorption should not change... which is good and might help in this objective.  

So here is where we pass in the PAR array to the photosynthesis routine:
https://github.com/NGEET/fates/blob/main/biogeophys/FatesPlantRespPhotosynthMod.F90#L582-L583

And here is where those arrays are scaled from a unit profile, to an absolute profile (by the forcing at the top of the canopy):
https://github.com/NGEET/fates/blob/main/biogeophys/EDSurfaceAlbedoMod.F90#L1249-L1257

I'm guessing its that second location in the code where you would want to scale up or down your PAR value

**Comment by liujia96** [comment]
> The radiation scattering over the depth of the canopy is not impacted by the magnitude of the downwelling radiation. So if you increased or decreased that magnitude, I think that the relative profiles of trasmittance and absorption should not change... which is good and might help in this objective.
> 
> So here is where we pass in the PAR array to the photosynthesis routine: https://github.com/NGEET/fates/blob/main/biogeophys/FatesPlantRespPhotosynthMod.F90#L582-L583
> 
> And here is where those arrays are scaled from a unit profile, to an absolute profile (by the forcing at the top of the canopy): https://github.com/NGEET/fates/blob/main/biogeophys/EDSurfaceAlbedoMod.F90#L1249-L1257
> 
> I'm guessing its that second location in the code where you would want to scale up or down your PAR value
Thank you for your patience and answers. So if I scale down the radiation to 50%, but at the same time do not change the photosynthesis. Can I modify the 'ipar' parameter. like bc_in(s)%solad_parb(ifp,ipar*2)*cpatch%fabd_sun_z(CL,ft,iv)

**Comment by rgknox** [comment]
If you wanted to decrease the the amount of PAR by 10% for instance, you could try something like this:

> bc_in(s)%solad_parb(ifp,ipar)*cpatch%fabd_sun_z(CL,ft,iv) / 1.1

But if I understand correctly, this only addresses half of what you wanted to do. It sounds like you would also need to increase the forcing by 10% as it applies to other aspects, like thermodynamics, and then decrease it back to the original value for photosynthesis (ie what we just did).  You also need to go into the CLM or ELM side of the code and modify the radiative forcing, I don't know exactly where I would do that though.  Also, you may run into radiation balance check errors.
